src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1852,13 +1837,85 @@
     </Field>
     <Field Name="Value" Type="ExpressionSyntax"/>
   </Node>
+
+  <AbstractNode Name="VariableComponentSyntax" Base="CSharpSyntaxNode">
+  </AbstractNode>
+  <Node Name="TypedVariableComponentSyntax" Base="VariableComponentSyntax">
+    <Kind Name="TypedVariableComponent"/>
+    <Field Name="Type" Type="TypeSyntax"/>
+    <Field Name="Designation" Type="VariableDesignationSyntax"/>
+  </Node>
+  <Node Name="TupleDeconstructionVariableComponentSyntax" Base="VariableComponentSyntax">
```

Comment:
I'm not a fan of either "Tuple" or "Deconstruction" in here.
`GroupedVariableComponentSyntax`?